data-raw/pwc_palettes.R

```diff
@@ -2,6 +2,11 @@
 
 pwc_palettes <- list(
   pwc_palette = pwc_cols("orange", "rose", "red", "tangerine", "yellow"),
+  # pwc_palette rearranged to maximize difference in colors:
+  pwc_palette2 = pwc_cols("orange", "yellow", "rose", "tangerine", "red"),
+  # pwc_palette rearranged and expanded for discrete use:
+  pwc_palette2_expanded = pwc_cols("orange", "orange1", "yellow", "medium_grey", "tangerine2", "rose",
+                                   "tangerine", "rose2", "grey", "red"),
```

Comment:
Ser ut som denne burde være default for scale_*_pwc_d-funksjonene, no?

Reply:
Enig, oppdatert med det nå!